package.json

```diff
@@ -155,7 +155,7 @@
     },
     {
       "path": "packages/react-instantsearch-core-next/dist/umd/ReactInstantSearchCore.min.js",
-      "maxSize": "28.25 kB"
+      "maxSize": "40 kB"
```

Comment:
We need to do some work to reduce the InstantSearch Core package bundle size.

Reply:
Have you already identified what the culprits are in InstantSearch Core?

Reply:
My bad, we used both ESM and CJS imports, resulting in a duplicated InstantSearch.js bundle being used here.

So we're now down from 40KB to 22 KB, which is smaller than the current RIS Core (28KB). It gives us room to implement all the other connectors!

![Bundle visualizer](https://user-images.githubusercontent.com/6137112/134890452-dcacfec4-8301-45aa-86d1-0ef966e8d368.png)